_episodes/l2-01-managing_contributions.md

```diff
@@ -3,6 +3,7 @@ title: "Managing contributions to code"
 teaching: 20
```

Comment:
We should make this 25 minutes now because it has the multiple branches section added
```suggestion
teaching: 25
```